cognite/client/data_classes/simulators/models.py

```diff
@@ -404,3 +399,273 @@ def _load_list(
         cls, resource: list[dict[str, Any]], cognite_client: CogniteClient | None = None
     ) -> list[SimulatorModelRevisionDependency]:
         return [cls._load(item, cognite_client) for item in resource]
+
+
+@dataclass
+class SimulatorModelRevisionDataObjectEdge(CogniteObject):
```

Comment:
for the props that fall under flowsheet, I think it's better to have prefix `SimulatorFlowsheet`, e.g.
- `SimulatorFlowsheetObjectEdge`
- `SimulatorFlowsheetObjectNode`
- `SimulatorFlowsheetProperty`
- `SimulatorFlowsheetPosition`
- `SimulatorFlowsheetGraphicalObject`